### ISSUE TITLE
after installing on my mac ventura 13.2.1 how do I run the iio oscilliscope application

### ISSUE THREAD

**Issue by ean123** [tfcollins/homebrew-formulae#20]

**Comment by tfcollins** [comment]
The main executable is "osc" so to run type "osc"

**Comment by ean123** [comment]
I type the "osc" command in the terminal, but I get "zsh: command not found: osc" error

**Comment by tfcollins** [comment]
Where do binaries get installed with your brew install? Did brew require additional linking after install?

**Comment by ean123** [comment]
yes homebrew was saved in /opt/homebrew then had to change the path for home brew to work
